packages/compass/.npmrc

```diff
@@ -0,0 +1,3 @@
+engine-strict=true
+registry=https://registry.npmjs.org/
+legacy-peer-deps=true
```

Comment:
Not sure if it's just github not showing it, but is this symlinked or copied over? If it's copied then if we bring .npmrc updates back (see https://github.com/mongodb-js/compass/pull/2270/files#r654220997), you'd need to do the same change in this file

Reply:
True that, i can add a copy step to hadron build, just wanted to avoid fighting with symlinks in windows, but i think the cache hack makes lots of sense there

Reply:
Ah, good to know! I wasn't aware that windows had issues with symlinking, I was sure it works fine there